diff -r -u4 SDL2-2.0.5/src/core/linux/SDL_evdev.c SDL2-2.0.5-mt/src/core/linux/SDL_evdev.c
--- SDL2-2.0.5/src/core/linux/SDL_evdev.c	2016-10-19 23:56:26.000000000 -0400
+++ SDL2-2.0.5-mt/src/core/linux/SDL_evdev.c	2017-08-09 23:11:39.121544373 -0400
@@ -56,8 +56,24 @@
 #ifndef SYN_DROPPED
 #define SYN_DROPPED 3
 #endif
 
+#define ABS_MT_SLOT             0x2f    /* MT slot being modified */
+#define ABS_MT_TOUCH_MAJOR      0x30    /* Major axis of touching ellipse */
+#define ABS_MT_TOUCH_MINOR      0x31    /* Minor axis (omit if circular) */
+#define ABS_MT_WIDTH_MAJOR      0x32    /* Major axis of approaching ellipse */
+#define ABS_MT_WIDTH_MINOR      0x33    /* Minor axis (omit if circular) */
+#define ABS_MT_ORIENTATION      0x34    /* Ellipse orientation */
+#define ABS_MT_POSITION_X       0x35    /* Center X touch position */
+#define ABS_MT_POSITION_Y       0x36    /* Center Y touch position */
+#define ABS_MT_TOOL_TYPE        0x37    /* Type of touching device */
+#define ABS_MT_BLOB_ID          0x38    /* Group a set of packets as a blob */
+#define ABS_MT_TRACKING_ID      0x39    /* Unique ID of initiated contact */
+#define ABS_MT_PRESSURE         0x3a    /* Pressure on contact area */
+#define ABS_MT_DISTANCE         0x3b    /* Contact hover distance */
+#define ABS_MT_TOOL_X           0x3c    /* Center X tool position */
+#define ABS_MT_TOOL_Y           0x3d    /* Center Y tool position */
+
 typedef struct SDL_evdevlist_item
 {
     char *path;
     int fd;
