--- ld64/src/3rd/helper.c.orig
+++ ld64/src/3rd/helper.c
@@ -33,11 +33,5 @@
 void __assert_rtn(const char *func, const char *file, int line, const char *msg)
 {
-#if defined(__FreeBSD__) || defined(__DragonFly__)
-    __assert(msg, file, line, func);
-#elif defined(__NetBSD__) || defined(__OpenBSD__) || defined(__CYGWIN__)
-    __assert(msg, line, file);
-#else
-    __assert(msg, file, line);
-#endif /* __FreeBSD__ */
+    __assert_fail(msg, file, line, func);
 }
